AA-427: Remove all mentions of 'aggregation' possible

forshtat opened the pull request:
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

ducthotran2010 committed:
Add ERC: Ownership Delegation and Context for ERC-721

Merged by EIP-Bot.

T1MOH593 committed:
Fix incorrect comment copied from previous function (#429)

SamWilsn committed:
Update erc-7588.md (#597)

socathie committed:
Website: Move ERC-7007 to Last Call

Merged by EIP-Bot.

alcueca committed:
Add ERC: Common Quote Oracle

Merged by EIP-Bot.

lukasrosario committed:
Update ERC-7677: Move to Last Call

Merged by EIP-Bot.

SamWilsn committed:
CI: Update stale.yml

Merged by EIP-Bot.

forshtat committed:
AA-427: Remove all mentions of 'aggregation' possible

kopy-kat committed:
Update ERC-7579: fix typo

Merged by EIP-Bot.

ernanirst committed:
Update ERC-7432: Update Specification

Merged by EIP-Bot.

Commit:
Add ERC: ERC-1155 Multi-Asset extension

Merged by EIP-Bot.

Vectorized committed:
Add ERC: Minimal Upgradeable Proxies

Merged by EIP-Bot.

CodeSandwich committed:
Add ERC: Puppet Proxy Contract

Merged by EIP-Bot.

dongri committed:
Add ERC: Transfer With Authorization

Merged by EIP-Bot.

Rjected committed:
Update ERC-3770: Fix broken links and formatting (#531)

* Update ERC-3770: Fix broken links and formatting

* use relative links only, for website

fulldecent committed:
Update ERC-7585: fix typo

Merged by EIP-Bot.

lukasrosario committed:
Update ERC-7677: postopgas callout

Merged by EIP-Bot.

gavfu committed:
Update ERC-7588: Move to Final

Merged by EIP-Bot.

ernanirst committed:
Update ERC-7432: Move to Last Call

Merged by EIP-Bot.

peersky committed:
Add ERC: Code Index

Merged by EIP-Bot.

peersky committed:
Add ERC: Composable Security Middleware Hooks

Merged by EIP-Bot.

chenly committed:
Update ERC-7627: Move to Review

Merged by EIP-Bot.

sullof committed:
Update ERC-7656: Move to Review

Merged by EIP-Bot.

chenly committed:
Update ERC-7628: Move to Review

Merged by EIP-Bot.

lukasrosario committed:
Update ERC-7677: Move to Review

Merged by EIP-Bot.

drortirosh commented on ERCS/erc-4337.md:
```diff
@@ -222,11 +206,9 @@ this bundler is supposed to track the `key` and `sequence` pair of the UserOpera
 ### Required entry point contract functionality
 
-There are 2 separate entry point methods: `handleOps` and `handleAggregatedOps`
+There is one main entry point method: `handleOps`.
```
```suggestion
The entry point method is `handleOps`, which handles an array of userOps
```

drortirosh commented on ERCS/erc-4337.md:
```diff
@@ -222,11 +206,9 @@ this bundler is supposed to track the `key` and `sequence` pair of the UserOpera
-* `handleOps` handles userOps of accounts that don't require any signature aggregator.
-* `handleAggregatedOps` can handle a batch that contains userOps of multiple aggregators (and also requests without any aggregator)
-* `handleAggregatedOps` performs the same logic below as `handleOps`, but it must transfer the correct aggregator to each userOp, and also must call `validateSignatures` on each aggregator before doing all the per-account validation.
+* `handleOps` handles userOps of accounts.
```
```suggestion

```

drortirosh commented on ERCS/erc-4337.md:
```diff
@@ -357,7 +312,7 @@ To add a UserOperation into the mempool (and later to add it into a bundle) we n
 For this purpose, a UserOperation is not allowed to access any information that might change between simulation and execution, such as current block time, number, hash etc.
 In addition, a UserOperation is only allowed to access data related to this sender address: Multiple UserOperations should not access the same storage, so it is impossible to invalidate a large number of UserOperations with a single state change.
-There are 3 special contracts that interact with the account: the factory (initCode) that deploys the contract, the paymaster that can pay for the gas, and a signature aggregator (described later)
+There are 2 special contracts that interact with the account: the factory (initCode) that deploys the contract, and the paymaster that can pay for the gas.
```
```suggestion
There are 2 special entity contracts that interact with the account: the factory (initCode) that deploys the contract, and the paymaster that can pay for the gas.
```

drortirosh commented on ERCS/erc-4337.md:
```diff
@@ -583,13 +531,11 @@ The result `SHOULD` be set to the **userOpHash** if and only if the request pass
-    * The `data` field SHOULD contain a `paymaster` or `aggregator` value, depending on the failed entity
-  * **code: -32505** - transaction rejected because paymaster (or signature aggregator) stake or unstake-delay is too low
-    * The `data` field SHOULD contain a `paymaster` or `aggregator` value, depending on the failed entity
+  * **code: -32504** - transaction rejected because paymaster is throttled/banned
```
irrelevant, as it is removed to an external ERC

forshtat replied:
Removed all mentions of error codes.

sullof committed:
Update ERC-7656: Fix wording, using service instead of account, when needed

Merged by EIP-Bot.

SamWilsn committed:
Remove polyfill. (#620)

The `polyfill.io` domain was taken over by a malicious actor, and now
injects code that redirects to unrelated websites.

References:

<https://sansec.io/research/polyfill-supply-chain-attack>
<https://blog.cloudflare.com/automatically-replacing-polyfill-io-links-with-cloudflares-mirror-for-a-safer-internet/>

Reported-By: Elliott Green <eg145@brighton.ac.uk>

forshtat committed:
Update ERCS/erc-4337.md

Co-authored-by: Dror Tirosh <dror.tirosh@gmail.com>

forshtat committed:
Update ERCS/erc-4337.md

Co-authored-by: Dror Tirosh <dror.tirosh@gmail.com>

forshtat committed:
Update ERCS/erc-4337.md

Co-authored-by: Dror Tirosh <dror.tirosh@gmail.com>

forshtat committed:
Remove mention of JSON-RPC API error codes from core ERC-4337 document

forshtat committed:
Update ERC-4337: Unused gas penalty only callGasLimit` and `paymasterPostOpGasLimit`

Merged by EIP-Bot.

shahafn commented on ERCS/erc-4337.md:
````diff
@@ -390,19 +344,15 @@ struct ReturnInfo {
-
 ```
 
+The `AggregatorStakeInfo` structure is further defined in [ERC-XXXX](link).
````
TODO add actual link

forshtat committed:
Merge branch 'master' into AA-427-extract-aggregation-remove

forshtat commented:
Closed in favour of https://github.com/ethereum/ERCs/pull/627
